src/mono/System.Private.CoreLib/src/System/Threading/ThreadPool.Mono.cs

```diff
@@ -10,7 +10,7 @@ namespace System.Threading
     public static partial class ThreadPool
     {
         [Conditional("unnecessary")]
-        internal static void ReportThreadStatus(bool isWorking)
+        internal static void ReportThreadStatusCore(bool isWorking)
```

Comment:
It seems this file can be deleted now. Previously, the implementation was there just for coreclr, now it's in `ThreadPool.Windows.cs` and `ThreadPool.Unix.cs`, so this would be redundant? Also I don't see any references to `ReportThreadStatusCore`.

Reply:
I meant "method" not "file", edited above

Reply:
Looks like the file was removed from the project

Reply:
My bad, I think I had meant to delete the file since this method is the only thing in it, thanks for pointing that out.